Source/Core/VideoBackends/OGL/TextureConverter.cpp

```diff
@@ -35,7 +35,7 @@ static GLuint s_texConvFrameBuffer[2] = {0,0};
 static GLuint s_srcTexture = 0; // for decoding from RAM
 static GLuint s_dstTexture = 0; // for encoding to RAM
 
-const int renderBufferWidth = 1024;
+const int renderBufferWidth = EFB_WIDTH * 4;
 const int renderBufferHeight = 1024;
```

Comment:
Yes... except that we also use this buffer for XFB encoding, which requires substantially more height.  If it matters, I can change it to use two separate buffers.